modules/rosa-edit-objects.adoc

```diff
@@ -158,7 +158,7 @@ $ rosa edit machinepool --cluster=<cluster_name> <machinepool_ID> [arguments]
 |Required: The name or ID (string) of the cluster to edit on which the additional machine pool will be edited.
 
 --enable-autoscaling
-|Enable autoscaling of compute nodes. Use this argument with the `--min-replicas` and `--max-replicas` arguments.
+|Enable autoscaling of compute nodes. Use this argument with the `--min-replicas` and `--max-replicas` arguments. To disable autoscaling, use `--enable-autoscaling=false` with the `--replicas` argument.
```

Comment:
I'd be clearer that you can both enable and disable autoscaling with this flag. I might add a separate row for _just_ disabling autoscaling, but if you want to cover all of the flags in one entry, it might look like this:

```suggestion
|Enable or disable autoscaling of compute nodes. To enable autoscaling, use this argument with the `--min-replicas` and `--max-replicas` arguments. To disable autoscaling, use `--enable-autoscaling=false` with the `--replicas` argument.
```

Reply:
It's using the same argument, just with a different value for disabling, so I can't justify a separate row in the table. So I went with the latter suggestion. Thanks!